modules/ROOT/pages/anypoint-support.adoc

```diff
@@ -0,0 +1,11 @@
+= Anypoint Support
```

Comment:
We need to clarify in this section that MuleSoft support needs to be contacted only to report issues or defects.

For assistance, guidance, professional services customizations and enhancement request please reach out to your MuleSoft Customer Success representative.

Reply:
Much better. Updated, and thank you for the info.